srv/components/provisioner/scripts/install.sh

```diff
@@ -260,13 +261,13 @@ config_local_salt()
 setup_repos_hosted()
 {
     repo=$1
-    for repo_file in $(grep -lE "cortx-storage.colo.seagate.com|file://" /etc/yum.repos.d/*.repo); do
+    for repo_file in $(grep -lE "cortx-storage.colo.seagate.com|baseurl=file:///" /etc/yum.repos.d/*.repo); do
```

Comment:
need to remove cortx-storage.colo.seagate.com and get the ip/hostname from the url provided to install.sh